### ISSUE TITLE
Error: could not detect installed version

### ISSUE THREAD

**Issue by ChristophRichner** [petersendev/azure-devops-dotnet-tool#7]
Hi there,

Just found and installed your cool Azure DevOps Extension. 

I'm using the following **yaml** build template on a **Hosted VS2017 Agent**

```
 - task: petersendev.dotnet-global-tool-installer.DotnetGlobalToolInstaller.DotnetGlobalToolInstaller@0
    displayName: '.NET Core Global Tool'
    inputs:
      name: MergeCoberturaXml
      versionSpec: 1.0.1
```

I'm facing this issue: **Error: could not detect installed version**

```
2018-11-04T20:41:14.7054828Z ##[section]Starting: .NET Core Global Tool
2018-11-04T20:41:14.7058123Z ==============================================================================
2018-11-04T20:41:14.7058206Z Task         : .NET Core Global Tool Installer
2018-11-04T20:41:14.7058253Z Description  : Installs a .NET Core Global Tool. Uses dotnet cli internally.
2018-11-04T20:41:14.7058315Z Version      : 0.1.6
2018-11-04T20:41:14.7058356Z Author       : Arne Petersen
2018-11-04T20:41:14.7058415Z Help         : 
2018-11-04T20:41:14.7058458Z ==============================================================================
2018-11-04T20:41:15.7811577Z [command]"C:\Program Files\dotnet\dotnet.exe" tool install --tool-path D:\a\_temp\2bece421-7f69-4199-90be-c919c352ff9f MergeCoberturaXml --version 1.0.1
2018-11-04T20:41:20.7905008Z You can invoke the tool using the following command: MergeCoberturaXml
2018-11-04T20:41:20.7905563Z Tool 'mergecoberturaxml' (version '1.0.1') was successfully installed.
2018-11-04T20:41:20.8891538Z Error: could not detect installed version
2018-11-04T20:41:20.8893142Z     at D:\a\_tasks\DotnetGlobalToolInstaller_58967a6f-2cc7-4a67-ba48-6595739428c2\0.1.6\dist\task.js:116:19
2018-11-04T20:41:20.8893398Z     at next (native)
2018-11-04T20:41:20.8894101Z     at D:\a\_tasks\DotnetGlobalToolInstaller_58967a6f-2cc7-4a67-ba48-6595739428c2\0.1.6\dist\task.js:7:71
2018-11-04T20:41:20.8894211Z     at __awaiter (D:\a\_tasks\DotnetGlobalToolInstaller_58967a6f-2cc7-4a67-ba48-6595739428c2\0.1.6\dist\task.js:3:12)
2018-11-04T20:41:20.8894300Z     at acquireTool (D:\a\_tasks\DotnetGlobalToolInstaller_58967a6f-2cc7-4a67-ba48-6595739428c2\0.1.6\dist\task.js:93:12)
2018-11-04T20:41:20.8895519Z     at D:\a\_tasks\DotnetGlobalToolInstaller_58967a6f-2cc7-4a67-ba48-6595739428c2\0.1.6\dist\task.js:70:71
2018-11-04T20:41:20.8895632Z     at next (native)
2018-11-04T20:41:20.8895719Z     at D:\a\_tasks\DotnetGlobalToolInstaller_58967a6f-2cc7-4a67-ba48-6595739428c2\0.1.6\dist\task.js:7:71
2018-11-04T20:41:20.8895794Z     at __awaiter (D:\a\_tasks\DotnetGlobalToolInstaller_58967a6f-2cc7-4a67-ba48-6595739428c2\0.1.6\dist\task.js:3:12)
2018-11-04T20:41:20.8895890Z     at getTool (D:\a\_tasks\DotnetGlobalToolInstaller_58967a6f-2cc7-4a67-ba48-6595739428c2\0.1.6\dist\task.js:33:12)
2018-11-04T20:41:20.8895991Z     at D:\a\_tasks\DotnetGlobalToolInstaller_58967a6f-2cc7-4a67-ba48-6595739428c2\0.1.6\dist\task.js:24:19
2018-11-04T20:41:20.8896065Z     at next (native)
2018-11-04T20:41:20.8896130Z     at D:\a\_tasks\DotnetGlobalToolInstaller_58967a6f-2cc7-4a67-ba48-6595739428c2\0.1.6\dist\task.js:7:71
2018-11-04T20:41:20.8896202Z     at __awaiter (D:\a\_tasks\DotnetGlobalToolInstaller_58967a6f-2cc7-4a67-ba48-6595739428c2\0.1.6\dist\task.js:3:12)
2018-11-04T20:41:20.8896292Z     at run (D:\a\_tasks\DotnetGlobalToolInstaller_58967a6f-2cc7-4a67-ba48-6595739428c2\0.1.6\dist\task.js:18:12)
2018-11-04T20:41:20.8896529Z     at Object.<anonymous> (D:\a\_tasks\DotnetGlobalToolInstaller_58967a6f-2cc7-4a67-ba48-6595739428c2\0.1.6\dist\task.js:136:1)
2018-11-04T20:41:20.8896729Z     at Module._compile (module.js:570:32)
2018-11-04T20:41:20.8896781Z     at Object.Module._extensions..js (module.js:579:10)
2018-11-04T20:41:20.8896847Z     at Module.load (module.js:487:32)
2018-11-04T20:41:20.8896894Z     at tryModuleLoad (module.js:446:12)
2018-11-04T20:41:20.8898239Z     at Function.Module._load (module.js:438:3)
2018-11-04T20:41:20.8898297Z     at Module.runMain (module.js:604:10)
2018-11-04T20:41:20.8973598Z ##[error]could not detect installed version
2018-11-04T20:41:20.9017825Z ##[section]Finishing: .NET Core Global Tool
```

Any idea what I'm doing wrong?

Thanks

**Comment by srollinet** [comment]
I had the same problem installing cake.tool a few days ago, but now it seems to work

**Comment by shep1987** [comment]
It was a case problem for me 
Didn't work
```
- task: petersendev.dotnet-global-tool-installer.DotnetGlobalToolInstaller.DotnetGlobalToolInstaller@0
  displayName: 'Install Git Version'
  inputs:
    name: GitVersion.Tool
    versionSpec: '4.0.1-beta1-50'
    includePrerelease: true
```
Worked
```
- task: petersendev.dotnet-global-tool-installer.DotnetGlobalToolInstaller.DotnetGlobalToolInstaller@0
  displayName: 'Install Git Version'
  inputs:
    name: gitversion.tool
    versionSpec: '4.0.1-beta1-50'
    includePrerelease: true
```

**Comment by Tiger333** [comment]
Also ran into the same problem and changing the casing resolved it for me too

**Comment by satheeshds** [comment]
Also run into the same problem.. solved by changing the case

**Comment by petersendev** [comment]
fixed in 0.2.1
